pages/querying/functions.mdx

```diff
@@ -157,12 +157,70 @@ This section contains the list of built-in query modules and their procedures.
 
 ### Schema
 
-The `schema` procedures return information about the database schema.
+The `schema` procedures return information about the database schema and can be used to assert constraints and indices exist in the database.
 
 <Callout type="info">
 You can execute these procedures on [graph projections, subgraphs or portions of the graph using the `project` function](/advanced-algorithms/run-algorithms#run-procedures-on-subgraph).
 </Callout>
 
+{<h4> <code> schema.assert() </code> </h4>}
+
+The `schema.assert()` is used to assert indices and constraints exist in Memgraph.
+
+{<h5> Input: </h5>}
+
+- `indices_map: (mgp.Map, required)`: Map of a label string to a list of string properties. For each property string, if it is empty, label-index will be created, 
+  otherwise label+property index will be created. If the provided list is empty, label index will be created.
+- `unique_constraints: (mgp.Map, required)`: Map of a label string a list of lists containing string properties. List of lists is required since unique constraints can be created on a combination
```

Comment:
uh this is soooo confusing
Did you wanted to say this:
Requires a map of key-value pairs where the key is a string representing a label and the value is a list of lists containing properties as strings.

Jedino ne razumijem zasto list of lists... can you give an example of this map?

Reply:
> Requires a map of key-value pairs where the key is a string representing a label and the value is a list of lists containing properties as strings.

Yes. Here we need a list of lists because one unique constraint is a pair => label: list<property>.
This is different from label+property indices because one label+property index is a pair => label: property.
So to define multiple unique constraints, you need to have a list of lists.